include/wrapper.sh

```diff
@@ -6,12 +6,14 @@ MISSING_VAR_MESSAGE="must be set"
 DEFAULT_AWS_REGION="us-west-2"
 DEFAULT_DATA_DIR="/opt/zookeeper/snapshots"
 DEFAULT_LOG_DIR="/opt/zookeeper/transactions"
+DEFAULT_ZK_ENSEMBLE_SIZE=0
```

Comment:
Zero is what it currently defaults to. Basically 0 means keep looking for other nodes until you think you are stable and enough have checked in.